common/src/main/java/tachyon/network/protocol/RPCBlockReadRequest.java

```diff
@@ -33,17 +34,19 @@ public RPCBlockReadRequest(long blockId, long offset, long length) {
     mLength = length;
   }
 
+  @Override
   public Type getType() {
     return Type.RPC_BLOCK_READ_REQUEST;
   }
 
   /**
-   * Decode the input {@link ByteBuf} into a {@link RPCBlockReadRequest} object and return it.
+   * Decodes the input {@link ByteBuf} into a {@link RPCBlockReadRequest} object and returns it.
    *
    * @param in the input {@link ByteBuf}
    * @return The decoded RPCBlockReadRequest object
    */
   public static RPCBlockReadRequest decode(ByteBuf in) {
+    Preconditions.checkNotNull(in);
```

Comment:
The other files have a blank line after this check. This should add the line, or the others should remove the blank line, to stay consistent.

Also, is there a reason to believe netty will pass in a null input? How would the new behavior be different from the existing behavior?